lib/rendition.js

```diff
@@ -271,12 +239,25 @@ class Rendition {
         return meta;
     }
 
+    changeInstructions(newInstructions) {
+        this._originalInstructions = this.instructions;
```

Comment:
I would do an Object.freeze (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) on this._originalInstructions here too, so I'm sure those are not mistakenly changed later.
```suggestion
        this._originalInstructions = this.instructions;
        Object.freeze(this._originalInstructions);
```